tests/hpc/powerman.pm

```diff
@@ -16,7 +16,7 @@
 # Maintainer: Matthias Griessmeier <mgriessmeier@suse.com>
 
 
-use base "hpcbase";
+use base "opensusebasetest";
```

Comment:
There's a call to `$self->enable_and_start` in line 48.

Reply:
Changed!